<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMetadatiDialog</class>
 <widget class="QDialog" name="EditMetadatiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>751</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Metadati</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Informazioni sui metadati</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_id_metadato">
         <property name="text">
          <string>ID Metadato</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="id_metadato">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="liv_gerarchico">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>series</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_liv_gerarchico">
         <property name="text">
          <string>Liv. Gerarchico</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_data_metadato">
         <property name="text">
          <string>Data Metadato</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="resp_metadato_nome">
         <property name="toolTip">
          <string>Inserire il nome e cognome dell'autore dello studio</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_resp_metadato_nome">
         <property name="text">
          <string>Resp. Metadato Nome</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="resp_metadato_email">
         <property name="toolTip">
          <string>Email dell'autore dello studio</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_resp_metadato_email">
         <property name="text">
          <string>Resp. Metadato Email</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="resp_metadato_sito">
         <property name="toolTip">
          <string>Inserire eventuale sito web dell'autore dello studio</string>
         </property>
         <property name="placeholderText">
          <string>https://example.com</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_resp_metadato_sito">
         <property name="text">
          <string>Resp. Metadato Sito</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QgsDateTimeEdit" name="data_metadato">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Informazioni sui dati</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_srs_dati">
         <property name="text">
          <string>SRS Dati</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="srs_dati">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>32633</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_formato">
         <property name="text">
          <string>Formato</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="formato">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>mapDigital</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_data_dato">
         <property name="text">
          <string>Data Dato</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_desc_dato">
         <property name="text">
          <string>Descrizione Dato</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="tipo_dato">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>vector</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_tipo_dato">
         <property name="text">
          <string>Tipo Dato</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="precisione">
         <property name="toolTip">
          <string>Precisione del dato (errore in metri)</string>
         </property>
         <property name="placeholderText">
          <string>0.2</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_precisione">
         <property name="text">
          <string>Precisione</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_genealogia">
         <property name="text">
          <string>Genealogia</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QgsDateTimeEdit" name="data_dato">
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTextEdit" name="desc_dato">
         <property name="toolTip">
          <string>Descrizione dei dati</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Studi di Microzonazione Sismica di livello (n) relativi al comune di ... </string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QTextEdit" name="genealogia">
         <property name="toolTip">
          <string>Descrizione della provenienza e dei processi di elaborazione dei dati</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Proprietà dei dati</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="proprieta_dato_nome">
         <property name="placeholderText">
          <string>Regione (nome)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_proprieta_dato_nome">
         <property name="text">
          <string>Nome del proprietario </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="proprieta_dato_email"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_proprieta_dato_email">
         <property name="text">
          <string>Email del proprietario</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="proprieta_dato_sito"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_proprieta_dato_sito">
         <property name="text">
          <string>Sito web del proprietario</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ruolo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>owner</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_ruolo">
         <property name="text">
          <string>Ruolo</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="contatto_dato_nome">
         <property name="placeholderText">
          <string>Regione (nome)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_contatto_dato_nome">
         <property name="text">
          <string>Nome del punto di contatto</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="contatto_dato_email"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_contatto_dato_email">
         <property name="text">
          <string>Email del punto di contatto</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="contatto_dato_sito"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_contatto_dato_sito">
         <property name="text">
          <string>Sito web del punto di contatto</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Classificazione, vincoli, copertura</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="label_keywords">
         <property name="text">
          <string>Keywords</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="keywords">
         <property name="text">
          <string>Microzonazione Sismica, Pericolosità Sismica</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_keywords_inspire">
         <property name="text">
          <string>Keywords Inspire</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="keywords_inspire">
         <property name="text">
          <string>Zone a rischio naturale, Geologia</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_limitazione">
         <property name="text">
          <string>Limitazione</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="limitazione">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>nessuna limitazione</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_vincoli_accesso">
         <property name="text">
          <string>Vincoli Accesso</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="vincoli_accesso">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>nessuno</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_vincoli_fruibilita">
         <property name="text">
          <string>Vincoli Fruibilita</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="vincoli_fruibilita">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>nessuno</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_vincoli_sicurezza">
         <property name="text">
          <string>Vincoli Sicurezza</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="vincoli_sicurezza">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>nessuno</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_scala">
         <property name="text">
          <string>Scala</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="scala">
         <property name="toolTip">
          <string>Scala nominale (solo denominatore, 10000 o scale più grandi come 5000)</string>
         </property>
         <property name="placeholderText">
          <string>5000</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_categoria_iso">
         <property name="text">
          <string>Categoria ISO</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="categoria_iso">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>geoscientificInformation</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_estensione_ovest">
         <property name="text">
          <string>Estensione Ovest</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="estensione_ovest">
         <property name="toolTip">
          <string>L'estensione geografica è calcolata automaticamente, modificare solo in caso di necessità</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_estensione_est">
         <property name="text">
          <string>Estensione Est</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="estensione_est">
         <property name="toolTip">
          <string>L'estensione geografica è calcolata automaticamente, modificare solo in caso di necessità</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_estensione_sud">
         <property name="text">
          <string>Estensione Sud</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="estensione_sud">
         <property name="toolTip">
          <string>L'estensione geografica è calcolata automaticamente, modificare solo in caso di necessità</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="estensione_nord">
         <property name="toolTip">
          <string>L'estensione geografica è calcolata automaticamente, modificare solo in caso di necessità</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_estensione_nord">
         <property name="text">
          <string>Estensione Nord</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Distribuzione</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label_formato_dati">
         <property name="text">
          <string>Formato Dati</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="formato_dati"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_distributore_dato_nome">
         <property name="text">
          <string>Distributore Dato Nome</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="distributore_dato_nome"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_distributore_dato_telefono">
         <property name="text">
          <string>Distributore Dato Telefono</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="distributore_dato_telefono"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_distributore_dato_email">
         <property name="text">
          <string>Distributore Dato Email</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="distributore_dato_email"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_distributore_dato_sito">
         <property name="text">
          <string>Distributore Dato Sito</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="distributore_dato_sito"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_url_accesso_dato">
         <property name="text">
          <string>URL Accesso Dato</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="url_accesso_dato"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_funzione_accesso_dato">
         <property name="text">
          <string>Funzione Accesso Dato</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="funzione_accesso_dato"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Attenzione: questi campi dovranno essere compilati dal soggetto distributore dei dati stessi (ad es. DPC, Regione, ecc.)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>img/Logo CNR-2010-Quadrato-ITA-high.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>img/IGAG-CMYK.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QDialogButtonBox" name="button_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="help_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>EditMetadatiDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>EditMetadatiDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
